src/spikeinterface/exporters/to_phy.py

```diff
@@ -15,17 +17,17 @@
 
 
 def export_to_phy(
-    waveform_extractor,
-    output_folder,
-    compute_pc_features=True,
-    compute_amplitudes=True,
-    sparsity=None,
-    copy_binary=True,
-    remove_if_exists=False,
-    peak_sign="neg",
-    template_mode="median",
-    dtype=None,
-    verbose=True,
+    waveform_extractor: WaveformExtractor,
+    output_folder: Union[str, Path],
```

Comment:
@h-mayorquin @DradeAW would it be better to use:

```
from __future__ import annotations

...
output_folder: str | Path

```

Reply:
I  like `str | Path` better.